background/web3.js

```diff
@@ -17,6 +17,8 @@
 import Config, { DEFAULT_CONFIG } from './config';
 import { TRANSPORT_UNINITIALIZED } from '../shared';
 
+const FAILURE = 'Failed to fetch';
```

Comment:
Should it be a Regex instead of a string comparison ? (eg. not caring about case)